dashboard/app/models/pd/enrollment.rb

```diff
@@ -103,6 +103,18 @@ def completed_survey?
     PEGASUS_DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: id).any?
   end
 
+  def local_summer?
+    workshop.course == Pd::Workshop::COURSE_CSP && workshop.subject == Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP
+  end
```

Comment:
Should this be in `workshop.rb`? It's really a property of the workshop, not enrollment.